# Errors in pattern matching functions

func() ->
  x.call(1 z)

